src/SceneManager.ts

```diff
@@ -1,12 +1,35 @@
 import * as THREE from 'three';
-import { GzAudioTopic } from './GzAudioTopic';
+import { AudioTopic } from './AudioTopic';
 import { Scene } from './Scene';
 import { SDFParser } from './SDFParser';
 import { Shaders } from './Shaders';
 import { Subscription } from 'rxjs';
 import { Topic } from './Topic';
 import { Transport } from './Transport';
 
+export class SceneManagerConfig {
```

Comment:
Since `SceneManagerConfig` represents the options that can be passed to the SceneManager constructor, it can be an interface with optional or default values.